template.tpl

```diff
@@ -441,36 +441,15 @@ ___TEMPLATE_PARAMETERS___
     "subParams": [
       {
         "type": "TEXT",
-        "name": "clientId",
-        "displayName": "Client ID",
+        "name": "conversionToken",
+        "displayName": "Conversion Token",
         "simpleValueType": true,
         "valueValidators": [
           {
             "type": "NON_EMPTY"
           }
-        ]
-      },
-      {
-        "type": "TEXT",
-        "name": "appSecret",
-        "displayName": "App Secret",
-        "simpleValueType": true,
-        "valueValidators": [
-          {
-            "type": "NON_EMPTY"
-          }
-        ]
-      },
-      {
-        "type": "TEXT",
-        "name": "refreshToken",
-        "displayName": "Refresh Token",
-        "simpleValueType": true,
-        "valueValidators": [
-          {
-            "type": "NON_EMPTY"
-          }
-        ]
+        ],
+        "notSetText": "Please generate a Conversion Token in your Reddit Ad account."
```

Comment:
```suggestion
        "notSetText": "Please generate a Conversion Token in your Reddit Ads account."
```

I think this aligns more with our own [documentation](https://business.reddithelp.com/helpcenter/s/article/Conversions-API).  
![Screenshot 2024-05-28 at 6 13 38 PM](https://github.com/reddit/reddit-ss-gtm-template/assets/136853654/5cf97bbc-2be8-4623-8b98-923e9994b044)